### ISSUE TITLE
Weapons list doesn't show in hud.

### ISSUE THREAD

**Issue by HLAG-Golden** [tmp64/BugfixedHL-Rebased#177]
Hello again, this is a problem that appeared in the last version of bugfixed. Basically, when pressing the numbers to see the weapons list they don't show, but when I activate the Rainbow HUD option it shows the weapons list.

![hud1](https://github.com/tmp64/BugfixedHL-Rebased/assets/117566992/1228ea95-55da-4f67-b527-851f30ef411f)
![hud2](https://github.com/tmp64/BugfixedHL-Rebased/assets/117566992/5c6d8908-ca8c-409f-b1b5-20eca0d727ab)

**Comment by Safety1st** [comment]
Same for me on both computers. Steam HL, W10. Nothing helps except enabling Rainbow HUD (thanks [HLAG-Golden](https://github.com/HLAG-Golden).

**Comment by tmp64** [comment]
When does this happen?

**Comment by HLAG-Golden** [comment]
It happens when I try to play, whether online or alone without the Rainbow HUD enabled.

**Comment by tmp64** [comment]
@Safety1st @HLAG-Golden Please, post your system information and check if the issue still happens on latest game update (2023-11-30, build 9897).
> The preferred and easiest way to get this information is from Steam's Hardware Information viewer from the menu (Help -> System Information). Once your information appears: right-click within the dialog, choose Select All, right-click again, and then choose Copy. Paste this information into your report, preferably in a code block.

**Comment by tmp64** [comment]
Looks like this only happens on Radeon RX 6000 so it's most likely an AMD driver bug. [It seems that downgrading the driver helps](https://github.com/ValveSoftware/halflife/issues/3408#issuecomment-1827319993)

**Comment by HLAG-Golden** [comment]
Hello sorry for the late response, in my case I have an Nvidia GTX 1050Ti. I installed the new update and I still have the same problem.

> Looks like this only happens on Radeon RX 6000 so it's most likely an AMD driver bug. [It seems that downgrading the driver helps](https://github.com/ValveSoftware/halflife/issues/3408#issuecomment-1827319993)

**Comment by Vladislav4KZ** [comment]
> Hello sorry for the late response, in my case I have an Nvidia GTX 1050Ti. I installed the new update and I still have the same problem.

I literally have the same GPU and have encountered the same bug.

Client version (taken from GitHub Actions artifacts):
![Снимок экрана 2024-08-29 230310](https://github.com/user-attachments/assets/9f1eaa22-01c1-41df-a9a3-0ddef458ac64)

Half-Life version:
```
Protocol version 48
Exe version 1.1.2.2/Stdio (valve)
Exe build: 12:23:38 Dec 22 2023 (9920)
```

**Comment by tmp64** [comment]
1. Check that none of the HUD colors are black/dark
2. Check if toggling "Enhanced sprite rendering" helps
3. Update your drivers (just in case)

\-

1. Проверь, что нигде не настроен черный/темный цвет
2. Попробуй поиграться с настройкой "Улучшенный рендеринг спрайтов"
3. Обнови видеодрайверы (на всякий случай)

**Comment by Vladislav4KZ** [comment]
> 1. Проверь, что нигде не настроен черный/темный цвет

Да, проблема была из-за цвета. Основной цвет интерфейса был чёрный.

Поменял цвет, и HUD стал нормально отображаться.

**Comment by Vladislav4KZ** [comment]
@Safety1st @HLAG-Golden change the main color from black to another. This bug is because of the black color.

This helped me solve the issue.
